config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,30 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I think govuk has a way of managing their security policies separately, so this file isn't necessary. See https://github.com/alphagov/government-frontend/pull/1811#discussion_r450932765

Reply:
:sparkles: